### PR TITLE
switch paragraphs, links and lists to digitalmarketplace-govuk-frontend's classes

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/RXXHKdGi

### CODE REVIEW

File: app/templates/notifications/user-research-consent.html [hunk 1]
```diff
@@ -68,15 +68,16 @@ <h1 class="govuk-heading-xl">
...
+      {% with 
+        label="Save changes", 
+        type="save", 
+        name="save_changes",
+        analytics = "tracking",
+        analytics_category = "user_research",
+        analytics_action = "subscribe" if not form.user_research_opt_in.data else "unsubscribe"
+      %}
+        {% include "toolkit/button.html" %}
+      {% endwith %}
```
Comment (lfdebrux) [inline app/templates/notifications/user-research-consent.html]: These changes are replacing a govukButton with a button from dmp-toolkit; maybe a git issue?

File: app/templates/notifications/user-research-consent.html [hunk 2]
```diff
@@ -68,15 +68,16 @@ <h1 class="govuk-heading-xl">
...
-      }) }}
+      {% with 
+        label="Save changes", 
+        type="save", 
+        name="save_changes",
+        analytics = "tracking",
+        analytics_category = "user_research",
+        analytics_action = "subscribe" if not form.user_research_opt_in.data else "unsubscribe"
+      %}
+        {% include "toolkit/button.html" %}
```
Comment (aliuk2012) [inline app/templates/notifications/user-research-consent.html]: not sure if this is meant to be in here

**Comment by risicle** [comment]
Damnit, retry that rebase...

**Comment by risicle** [comment]
Well caught - I should expand my grepping.

**Pull request merged**